client/src/components/layouts/Booking/Booking.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import InfiniteCalendar from 'react-infinite-calendar';
+import 'react-infinite-calendar/styles.css';
+import { Box } from '@material-ui/core';
+import RadioButton from '../BeforeVisit/Radio';
+
+const today = new Date();
+const lastWeek = new Date(
+  today.getFullYear(),
+  today.getMonth(),
+  today.getDate() - 7,
+);
+
+const Booking = (props) => {
+  const [selectedDate, setSelectedDate] = useState('');
+
+  const {
+    formField: { appointmentTime },
+  } = props;
+
+  const timeSlots = [
+    {
+      value: `${selectedDate} 10:00`,
+      label: '10:00 - 12:00',
+    },
+    {
+      value: `${selectedDate} 12:00`,
+      label: '12:00 - 14:00',
+    },
+    {
+      value: `${selectedDate} 14:00`,
+      label: '14:00 - 16:00',
+    },
+    {
+      value: `${selectedDate} 16:00`,
+      label: '16:00 - 18:00',
+    },
+  ];
+
+  const displayTimeSlots = (date) => {
+    setSelectedDate(
+      `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`,
+    );
+  };
+  return (
+    <Box display="flex" flexDirection="column">
+      <InfiniteCalendar
+        width={400}
+        height={300}
+        selected={today}
+        disabledDays={[0, 6]}
+        minDate={lastWeek}
+        onSelect={displayTimeSlots}
+        theme={{
```

Comment:
some inline style out there :eyes:  
![image](https://user-images.githubusercontent.com/56412800/87531028-cbd12a00-c699-11ea-83c5-e8427bd52e31.png)

Reply:
I will leave this part for you 😜 when you make an adjustments for the whole style.

Reply:
kidding right!! and give you the change to comment another "HOW DARE YOUUU" :open_mouth: